condition.go

```diff
@@ -117,3 +120,52 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+func buildLikeCmp(d Dialect, buf Buffer, pred string, column string, value interface{}) error {
+	if value == nil {
+		return fmt.Errorf("Column %s %s nil is Not Supported", column, pred)
+	}
+
+	v := reflect.ValueOf(value)
+	switch {
+	case v.Type() == reflect.TypeOf([]rune{}):
+		return buildCmp(d, buf, pred, column, string(value.([]rune)))
+
+	case v.Kind() == reflect.TypeOf([0]rune{}).Kind():
```

Comment:
I think it would be better to avoid type checkin here.  Because there is a lot of cases where value can be converted to proper string value. 
there is also implemented logic to handle different interfaces and properly convert them into valid database value. (method encodePlaceholder)
In any case if user pass not valid value to your method, it will see result as failed sql query.

If you want to strict check that passed value is valid, I think it would be better to rewrite this function like this:

```golang

	v := reflect.ValueOf(value)
	switch v.Kind() {
	case reflect.String:
		// pass as is
		return buildCmp(d, buf, pred, column, value)
	case reflect.Ptr, reflect.Interface:
		return buildLikeCmp(d, buf, pred, column, v.Elem.Interface())
	case reflect.Slice:
		switch v.Type().Elem().Kind() {
		case reflect.Uint8: // bytes
			// interpolator will handle this case
			return buildCmp(d, buf, pred, column, value)
		case reflect.Int32: // rune
			// need to convert into string
			// I am not sure that it is good idea to convert rune here
			// I think it would be better to ask user to convert into string
			return buildCmp(d, buf, pred, column, string(value([]rune)))
		}
		fallthrough
	default:
		return fmt.Errorf("Column %s %s Invalid Value, string expected", column, pred)
	}
}
```